\documentclass[12pt]{article}

% Load packages
\usepackage{url}  % Formatting web addresses
\usepackage{ifthen}  % Conditional
\usepackage{multicol}   %Columns
\usepackage[utf8]{inputenc} %unicode support
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{epsfig}
\usepackage{epstopdf}
\usepackage{graphicx}
\usepackage{cite}
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{mathtools}
\usepackage[margin=0.1pt,font=footnotesize,labelfont=bf]{caption}
\usepackage{setspace}
%\usepackage{longtable}
\usepackage{colortbl}
%\usepackage{palatino,lettrine}
%\usepackage{times}
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails
\usepackage[wide]{sidecap}
%\usepackage[authoryear,round,comma,sort&compress]{natbib}
\usepackage[square,sort,comma,numbers,sort&compress]{natbib}
%\usepackage[authoryear,round]{natbib}
\usepackage{supertabular}
\usepackage{simplemargins}
\usepackage{fullpage}
\usepackage{comment}
\usepackage{lineno}
%\usepackage{chicago}
\usepackage{textcomp}
\usepackage{multirow}
\usepackage{amsmath}
%\usepackage{textgreek}
%\usepackage[linesnumbered,lined,boxed,commentsnumbered]{algorithm2e}
\DeclareMathOperator*{\argmin}{\arg\!\min}

%\usepackage{algorithm2e}
%\usepackage{algpseudocode}
%\usepackage[space]{cite}
\urlstyle{rm}

%\textwidth = 6.50 in
%\textheight = 9.5 in
%\oddsidemargin =  0.0 in
%\evensidemargin = 0.0 in
%\topmargin = -0.50 in
%\headheight = 0.0 in
%\headsep = 0.25 in
%\parskip = 0.15in
%\linespread{1.75}
\doublespace

%\bibliographystyle{chicago}
\bibliographystyle{plos2009}

\makeatletter
\renewcommand\subsection{\@startsection
	{subsection}{2}{0mm}
	{-0.05in}
	{-0.5\baselineskip}
	{\normalfont\normalsize\bfseries}}
\renewcommand\subsubsection{\@startsection
	{subsubsection}{2}{0mm}
	{-0.05in}
	{-0.5\baselineskip}
	{\normalfont\normalsize\itshape}}
\renewcommand\section{\@startsection
	{subsection}{2}{0mm}
	{-0.2in}
	{0.05\baselineskip}
	{\normalfont\large\bfseries}}
\renewcommand\paragraph{\@startsection
	{paragraph}{2}{0mm}
	{-0.05in}
	{-0.5\baselineskip}
	{\normalfont\normalsize\itshape}}
\makeatother

%Review style settings
%\newenvironment{bmcformat}{\begin{raggedright}\baselineskip20pt\sloppy\setboolean{publ}{false}}{\end{raggedright}\baselineskip20pt\sloppy}

%Publication style settings

% Single space'd bib -
\setlength\bibsep{0pt}

\renewcommand{\rmdefault}{phv}\renewcommand{\sfdefault}{phv}
\newcommand{\norm}[1]{\left\lVert#1\right\rVert}

% Change the number format in the ref list -
\renewcommand{\bibnumfmt}[1]{#1.}

% Change Figure to Fig.
\renewcommand{\figurename}{Fig.}

% Begin ...
\begin{document}
\begin{titlepage}
{\par\centering\textbf{\Large {Toward a Genome Scale Dynamic Model of Cell-Free Protein Synthesis in \emph{Escherichia~coli}}}}
\vspace{0.05in}
{\par \centering \large{Nicholas Horvath, Michael Vilkhovoy, Joseph Wayman, Kara Calhoun$^{1}$, James Swartz$^{1}$ and Jeffrey D. Varner$^{*}$}}
\vspace{0.10in}
{\par \centering {Robert Frederick Smith School of Chemical and Biomolecular Engineering}}
{\par \centering {Cornell University, Ithaca NY 14853}}
\vspace{0.1in}
{\par \centering {$^{1}$School of Chemical Engineering}}
{\par \centering {Stanford University, Stanford, CA 94305}}
{\par \centering \textbf{Running Title:}~Dynamic modeling of cell-free protein synthesis}
\vspace{0.1in}
{\par \centering \textbf{To be submitted:}~\emph{Scientific~Reports}}
\vspace{0.5in}
{\par \centering $^{*}$Corresponding author:}
{\par \centering Jeffrey D. Varner,}
{\par \centering Professor, Robert Frederick Smith School of Chemical and Biomolecular Engineering,}
{\par \centering 244 Olin Hall, Cornell University, Ithaca NY, 14853}
{\par \centering Email: jdv27@cornell.edu}
{\par \centering Phone: (607) 255 - 4258}
{\par \centering Fax: (607) 255 - 9166}
\end{titlepage}
\date{}
\thispagestyle{empty}
\pagebreak
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Abstract}
Cell-free protein expression systems have become widely used in systems and synthetic biology.
In this study, we developed an ensemble of dynamic \textit{E. coli} cell-free protein synthesis (CFPS) models.
Model parameters were estimated from measurements of glucose, organic acids, energy species, amino acids and the protein product, chloramphenicol acetyltransferase (CAT).
The ensemble described all of the training data, especially the central carbon metabolism.
To gauge the performance of the cell-free reaction, we compared the observed CAT carbon yield, with the maximum theoretical CAT carbon yield calculated using sequence specific flux balance analysis.
The observed CAT yield was 45\% of the maximum theoretical yield, suggesting CAT production could be further optimized.
%The metabolic flux distribution predicted by the dynamic model and flux balance analysis were significantly different.
The ensemble of dynamic models predicted the majority of carbon flux was routed through glycolysis and the TCA cycle, while flux balance analysis predicted significant flux through the Entner-Doudoroff pathway.
Local and global sensitivity analysis suggested CAT production was most sensitive to parameters and initial conditions directly associated with CAT synthesis, as well as GTP/GMP synthesis, amino acid synthesis, and to a lesser extent amino acid initial conditions.
On the other hand, CAT production was robust to allosteric control parameters and the initial conditions of glucose and oxygen.
Taken together, we presented the first dynamic model of \textit{E. coli} cell-free protein synthesis.
This study provides a foundation for genome-scale, dynamic modeling of cell-free \textit{E. coli} protein synthesis.

\vspace{0.1in}
{\noindent \textbf{Keywords:}~Biochemical engineering, systems biology, cell-free protein synthesis}

\pagebreak

\setcounter{page}{1}

%In this study, we present a framework for dynamic, cell-free metabolic modeling, integrating a simple logical description of regulation with traditional enzyme kinetics.
%Using this framework, we have constructed an ensemble of models for production of chloramphenicol acetyltransferase in a cell-free \textit{E. coli} system.
%Our ensemble fits measurements of glucose, chloramphenicol acetyltransferase, organic acids, and energy species, but fails to capture much of the amino acid dynamics in the dataset.

% Uncomment in production -
\linenumbers


\section*{Introduction}

Cell-free systems offer many advantages for the study, manipulation and modeling of metabolism compared to \textit{in vivo} processes.
Central amongst these advantages is direct access to metabolites and the microbial biosynthetic machinery without the interference of a cell wall.
This allows us to control as well as interrogate the chemical environment while the biosynthetic machinery is operating, potentially at a fine time resolution.
Second, cell-free systems also allow us to study biological processes without the complications associated with cell growth.
Cell-free protein synthesis (CFPS) systems are arguably the most prominent examples of cell-free systems used today \citep{Jewett:2008aa}.
However, CFPS is not new; CFPS in crude \textit{E. coli} extracts has been used since the 1960s to explore fundamentally important biological mechanisms \citep{MATTHAEI:1961aa,NIRENBERG:1961aa}.
Today, cell-free systems are used in a variety of applications ranging from therapeutic protein production \citep{Lu:2014aa} to synthetic biology \citep{Hodgman:2012aa}.
Interestingly, many of the challenges confronting genome-scale kinetic modeling can potentially be overcome in a cell-free system.
For example, there is no complex transcriptional regulation to consider, transient metabolic measurements are easier to obtain, and we no longer have to consider cell growth.
Thus, cell-free operation holds several significant advantages for model development, identification and validation. Theoretically, genome-scale cell-free kinetic models may be possible for industrially important organisms,
such as \textit{E. coli} or \textit{B.~subtilis}, if a simple, tractable framework for integrating allosteric regulation with enzyme kinetics can be formulated.

Mathematical modeling has long contributed to our understanding of metabolism.
Decades before the genomics revolution, mechanistically, structured metabolic models arose from the desire to predict microbial phenotypes resulting from changes in intracellular or extracellular states \citep{1976_fredrickson_BiotechBioeng}.
The single cell \textit{E. coli} models of Shuler and coworkers pioneered the construction of large-scale, dynamic metabolic models that incorporated multiple, regulated catabolic and anabolic pathways constrained by experimentally determined kinetic parameters \citep{1984_domach_shuler_BiotechBioeng_01}.
Shuler and coworkers generated many single cell kinetic models, including single cell models of eukaryotes \citep{1989_steinmeyer_shuler_ChemEngSci,1992_wu_shuler_AnnNYAcadSci}, minimal cell architectures \citep{2004_castellanos_shuler_PNAS}, as well as DNA sequence based whole-cell models of \textit{E. coli} \citep{2008_atlas_shuler_IETSysBio}.
Conversely, highly abstracted kinetic frameworks, such as the cybernetic framework, represented a paradigm shift, viewing cells as growth-optimizing strategists \citep{1985_dhurjati_ramkrishna_tsao_BiotechBioeng}.
Cybernetic models have been highly successful at predicting metabolic choice behavior, e.g., diauxie behavior \citep{1986_kompala_ramkrishna_tsao_BiotechBioeng}, steady-state multiplicity \citep{2012_kim_ramkrishna_BiotechProg}, as well as the cellular response to metabolic engineering modifications \citep{1999_varner_ramkrishna_MetaEng}.
Unfortunately, traditional, fully structured cybernetic models also suffer from an identifiability challenge, as both the kinetic parameters and an abstracted model of cellular objectives must be estimated simultaneously.
However, recent cybernetic formulations from Ramkrishna and colleagues have successfully treated this identifiability challenge through elementary mode reduction~\cite{2009_song_ramkrishna_BiotechBioeng,Song:2011aa}.

In the post genomics world, large-scale stoichiometric reconstructions of microbial metabolism popularized by static, constraint-based modeling techniques such as flux balance analysis (FBA) have become standard tools \citep{2012_lewis_palsson_NatRevMicrobio}.
Since the first genome-scale stoichiometric model of \textit{E. coli}, developed by Edwards and Palsson \citep{2000_edwards_palsson_PNAS}, well over 100 organisms, including industrially important prokaryotes such as \textit{E. coli} \citep{Feist:2007aa} or \textit{B. subtilis} \citep{Oh:2007aa}, are now available \citep{2009_feist_palsson_NatRevMicrobio}.
Stoichiometric models rely on a pseudo-steady-state assumption to reduce unidentifiable genome-scale kinetic models to an underdetermined linear algebraic system, which can be solved efficiently even for large systems.
Traditionally, stoichiometric models have also neglected explicit descriptions of metabolic regulation and control mechanisms, instead opting to describe the choice of pathways by prescribing an objective function on metabolism.
Interestingly, similar to early cybernetic models, the most common metabolic objective function has been the optimization of biomass formation \citep{2002_ibarra_edwards_palsson_Nat}, although other metabolic objectives have also been estimated \citep{2007_schuetz_sauer_MolSysBio}.
Recent advances in constraint-based modeling have overcome the early shortcomings of the platform, including capturing metabolic regulation and control \citep{2013_hyduke_lewis_palsson_MolBioSys}.
Thus, modern constraint-based approaches have proven extremely useful in the discovery of metabolic engineering strategies and represent the state of the art in metabolic modeling \citep{2013_mccloskey_palsson_feist_MolSysBio, 2012_zomorrodi_maranas_MetaEng}.
However, genome-scale kinetic models of industrial important organisms such as \textit{E. coli} have yet to be constructed.

In this study, we developed an ensemble of \textit{E. coli} cell-free protein synthesis (CFPS) models using the hybrid cell-free modeling approach of Wayman et al [REFHERE].
Model parameters were estimated from measurements of glucose, organic acids, energy species, amino acids and the protein product, chloramphenicol acetyltransferase (CAT).
The ensemble described the training data, with the exception of some of the amino acid dynamics.
To gauge the performance of the cell-free reaction, we compared the observed CAT carbon yield, with the maximum theoretical CAT carbon yield calculated using sequence specific flux
balance analysis. The observed CAT yield was 45\% of the maximum theoretical yield, suggesting CAT production could be further optimized.
The metabolic flux distribution predicted by the dynamic model and flux balance analysis were significantly different.
The ensemble of dynamic models predicted the majority of carbon flux was routed through glycolysis and the TCA cycle,
while flux balance analysis predicted significant flux through the Entner-Doudoroff pathway.
Local and global sensitivity analysis suggested CAT production was most sensitive to parameters and initial conditions directly associated with CAT synthesis, as well as
GTP/GMP synthesis, amino acid synthesis, and to a lesser extent amino acid initial conditions.
On the other hand, CAT production was robust to allosteric control parameters and the initial conditions of glucose and oxygen.
Taken together, we presented the first dynamic model of \textit{E. coli} cell-free protein synthesis.
We integrated traditional kinetics with a logical rule-based description of allosteric control to simulate a comprehensive CFPS dataset.
This study provides a foundation for genome-scale, dynamic modeling of cell-free \textit{E. coli} protein synthesis.

% In this study, we present an effective biochemical network modeling framework for building dynamic cell-free metabolic models.
% The key innovation of our approach is the seamless integration of simple effective rules encoding complex regulation with traditional kinetic pathway modeling.
% This integration allows the description of complex regulatory interactions in the absence of specific mechanistic information.
% The regulatory rules are easy to understand, easy to formulate and do not rely on overarching theoretical abstractions or restrictive assumptions.
% Using this approach, we have built an ensemble of models that predict an experimental dataset for CAT production in a cell-free \textit{E. coli} system.
% We calculated the CAT yield for the ensemble at 45\% of the theoretical maximum according to constraint-based modeling, and determined through sensitivity analysis that CAT production was most sensitive to the parameters associated with CAT synthesis and the synthesis of GTP, GMP, and amino acids.
% While only an initial proof-of-concept, the framework presented here could be an important first step toward genome-scale cell-free kinetic modeling of the biosynthetic capacity of industrially important organisms.

\clearpage

\section*{Results}

\subsection*{Estimation of an ensemble of cell-free protein synthesis models.}
We used the hybrid cell-free modeling framework of Wayman et al. to simulate the production of a model protein [REFHERE].
The cell-free \textit{E. coli} metabolic model was constructed by removing the growth-associated processes from the model of Palsson and coworkers \cite{2000_edwards_palsson_PNAS}, and by adding reactions for the synthesis of chloramphenicol acetyltransferase (CAT), a model protein for which we have a comprehensive training dataset \cite{2005_calhoun_BiotechnologyProgress}.
Thus, the model described core central carbon metabolism (glycolysis, pentose phosphate, Enter-Doudoroff, TCA cycle), as well as the synthesis of energy species, amino acids biosynthesis and degradation, and biosynthesis of the CAT protein.
An ensemble of model parameters was estimated from dynamic measurements of glucose, CAT, organic acids (pyruvate, lactate, acetate, succinate, malate), energy species (A(x)P, G(x)P, C(x)P, U(x)P), and 18 of the 20 proteinogenic amino acids.
We generated an ensemble of N = 100 diverse parameter sets by minimizing the error between the training dataset and the metabolite concentrations predicted by the model, and choosing sets so as to minimize the Pearson correlation coefficient between sets in the ensemble.
We also defined the single set with lowest cost function as our best-fit set.

The ensemble of models captured the time evolution of cell-free CAT biosynthesis (Fig.~\ref{fig:CarbonBoth} - \ref{fig:Amino}).
The central carbon metabolism (Fig.~\ref{fig:CarbonBoth}, top), energy species (Fig.~\ref{fig:Energy}), and amino acids (Fig.~\ref{fig:Amino}) were all well captured by the ensemble and the best-fit set.
Allosteric control was seen to be particularly important to the dynamics of the organic acid intermediates, as without it several of the measurements are not captured by the ensemble or the best-fit set (Fig.~\ref{fig:CarbonBoth}, bottom).

\subsection*{Sensitivity analysis}
We performed a local sensitivity analysis to determine the network reactions with the greatest effect on protein production and overall system state.
%CAT production was most sensitive to the CAT synthesis reaction, oxidative phosphorylation activity, and alanine synthesis, as well as various reactions in glycolysis, pentose phosphate, the TCA cycle, and amino acid synthesis and degradation (Fig.~\ref{fig:Sensitivity}, top).
CAT production was most sensitive to the CAT synthesis reaction, oxidative phosphorylation activity, and alanine synthesis (Fig.~\ref{fig:Sensitivity}, top, section A).
The 16 next most important reactions to CAT production (section B) came from various pathways across the network: 4 each from glycolysis, the TCA cycle, and amino acid synthesis/degradation; 2 from pentose phosphate; and 1 each from the Entner-Doudoroff pathway and the energy species reactions.
The pairwise sensitivities (off-diagonal elements) are often quite different from the corresponding first-order sensitivities (diagonal elements), and lead to some interesting outcomes.
For example, glutamine synthesis and arginine degradation are both among the most important reactions to CAT production (they rank 5th and 10th, respectively).
This is likely because they both affect the sensitive glutamine-glutamate balance; glutamine synthesis consumes glutamate, while arginine degradation produces it.
However, when both are perturbed, their combined effect on the model is very low, as the respective consumption and production of glutamate cancel out.

The system state as a whole was most sensitive to glucose uptake via GTP and the forward reaction of lactate dehydrogenase (pyruvate being consumed to produce lactate) (Fig.~\ref{fig:Sensitivity}, bottom, section F).
The 30 next most important reactions to the system state (section G) came from various pathways across the network: 8 from amino acid synthesis/degradation; 6 from glycolysis; 4 from TCA; 2 each from pentose phosphate, Entner-Doudoroff, energy species reactions, NAD species reactions, and small molecule transport; 1 from oxidative phosphorylation; and 1 pyrophosphatase.
The system state has even more pairwise sensitivities that differ from the corresponding first-order sensitivities and stand out as significant.
For example, the first-order effect of alanine synthesis is very high; it consumes both pruvate and glutamate, two key species in the network.
In addition, a handful of alanine synthesis pairwise sensitivities are high.
However, there are enough reactions that, when paired with alanine synthesis, have little effect on the model; malic enzyme is one of these, as it produces the pyruvate that alanine synthesis consumes.
Thus, the total-order alanine synthesis sensitivity is low, placing it at the very bottom of section I.
Another interesting result of this analysis is seen in the intersection of sections F and G with section J.
The 53 reactions in section J were turned off in the best-fit set (rate constants were set to 0); therefore, the perturbation of these reactions (multiplying the rate constant by 1.01) had no actual effect on the model.
Thus, all pairwise sensitivities with reactions in section J can essentially be considered first-order sensitivities for the other reactions.
Interestingly, the reactions in section F and several in section G showed most of thier highest sensitivities when paired with the "non-effects" of section J.
Of these, three involved pyruvate, strengthening its role as a key metabolite; the others were glucose uptake via GTP and CTP, fumarate reductase, and SO$_4$ transport.
This suggests that these reactions' effects on the model were canceled out or lessened by most other reactions, but were of course not affected by the reactions in section J.

\subsection*{Maximum theoretical CAT yield showed CFPS can be optimized.}
We calculated the carbon yield of CAT production for our experimental data and our best-fit parameter set as a function of the initial and final concentrations and the carbon numbers of CAT, glucose, and amino acids.
%Arginine and glutamate were excluded due to not being present in the training dataset.
The experimental data displayed a CAT yield of 0.0865, while the best-fit parameter set displayed a CAT yield of 0.0871.
While the model ensemble described the experimental data, it was unclear whether the performance of the CFPS system was optimal.
To address this question, we used ssFBA in combination with the cell-free metabolic network and a detailed promoter model under a T7 polymerase to compute the maximum theoretical carbon yield.
We first validated the ssFBA approach by comparing an ensemble of simulated versus measured concentrations of CAT over a one hour period (Fig.~\ref{fig:CATProdYield}A). 
The ensemble of 100 sets captured the CAT concentration profile which was generated by sampling RNA polymerase levels, ribosome levels and elongation rates in a physiological range.
We then used sequence-specific FBA to calculate a theoretical maximum CAT yield under three different cases: unconstrained, constrained by transcription/translation rates, and constrained by transcription/translation rates and measurements (Fig.~\ref{fig:CATProdYield}B).
The theoretical maximum carbon yield of CAT was 0.35 $\pm$ 0.006 for an unconstrained case and 0.225 $\pm$ 0.03 for the transcription and translation constrained case.
Thus, the experimental dataset and best-fit parameter set each produced CAT at 25\% of the theoretical maximum and 38\% of a theoretical physiological case.
However, the case constrained by experimental data showed a carbon yield of 0.062 $\pm$ 0.02, similar to the experimental yield. 
In comparing the flux distributions between the unconstrained and constrained cases (Fig.~\ref{fig:Network}), the constrained cases heavily utilized the first step in the pentose phosphate pathway to generate NADPH.
In these cases the majority of the flux continued through the Entner–Doudoroff pathway, whereas in the unconstrained case the majority of flux traveled through glycolysis.
In all cases, the energy source came primarily from oxidative phosphorylation, as well as partly from the TCA cycle.
In the TX/TL constrained case, there was a high flux through fumerate dehydrogenase from aspartic acid uptake, whereas in the unconstrained and most constrained cases, acetate and lactate accumulation occurred.
This shows that the system is producing NADH through lactate dehydrogenase as well as through pyridine nucleotide transhydrogenase (\textit{pntAB}) to supply enough NADH for oxidative phosphorylation.
As a result, high oxidative phosphorylation activity relative to our cell-free system leads to an acetate overflow.
This suggests that there is potential for increasing CAT production by reducing this diversion of carbon.
To simulate potential knockouts, we constrained the specific glucose and amino acid uptake rates to the same values as simulated with no knockouts.
In an ssFBA simulation with constrained TX/TL rates, knocking out the \textit{gnd} reaction decreases flux of acetate production but increases flux through \textit{pntAB}, which is responsible for regenerating NADPH.
The simulation showed carbon was diverted toward lactate; however, since CAT production is constrained by the translation rate, we expected no increase in CAT production.
The decrease in acetate production is promising as a mechanism to increase CAT yield. 
A second simulation with a knockout of \textit{gnd} and phosphate acetyltransferase showed carbon being diverted toward lactate and succinate; however, it required a higher flux through oxidative phosphorylation and the TCA cycle to meet the energetic needs of the system.   

\clearpage

\section*{Discussion}

%The discussion has three (sometimes four) paragraphs:
%\begin{enumerate}
%	\item{\textbf{First~paragraph}: Present a modified version of the last paragraph of the introduction. In this study, [...]. Taken together, [killer statement]}
%	\item{\textbf{Second~paragraph}: Contrast the key findings of the study with other computational/experimental studies}
%	\item{\textbf{Third~paragraph}: Present future directions. If you had more time, what would like to do? Highlight the key shortcomings of the approach and how will we address them in the future.
%	In this case, we will have a scaling issue if we extend to genome scale. We should extend to dynamic cases, and we need to experimentally validate the findings.}
%\end{enumerate}

In this study, we developed an ensemble of \textit{E. coli} cell-free protein synthesis (CFPS) models using the hybrid cell-free modeling approach of Wayman et al [REFHERE].
Model parameters were estimated from measurements of glucose, organic acids, energy species, amino acids and the protein product, chloramphenicol acetyltransferase (CAT).
The ensemble described the training data, with the exception of some of the amino acid dynamics.
To gauge the performance of the cell-free reaction, we compared the observed CAT carbon yield with the maximum theoretical CAT carbon yield calculated using sequence specific flux balance analysis.
The observed CAT yield was 25\% of the maximum theoretical yield, suggesting CAT production could be further optimized.
The metabolic flux distribution predicted by the dynamic model and flux balance analysis were significantly different.
The ensemble of dynamic models predicted the majority of carbon flux was routed through glycolysis and the TCA cycle, while flux balance analysis predicted significant flux through the Entner-Doudoroff pathway.
Local and global sensitivity analysis suggested CAT production was most sensitive to parameters and initial conditions directly associated with CAT synthesis, as well as GTP/GMP synthesis, amino acid synthesis, and to a lesser extent amino acid initial conditions.
On the other hand, CAT production was robust to allosteric control parameters and the initial conditions of glucose and oxygen.
% Taken together, we presented the first dynamic model of \textit{E. coli} cell-free protein synthesis.
% We integrated traditional kinetics with a logical rule-based description of allosteric control to simulate a comprehensive CFPS dataset.
% This study provides a foundation for genome-scale, dynamic modeling of cell-free \textit{E. coli} protein synthesis.

In comparing the theoretical maximum carbon yield of CAT from ssFBA predictions to the kinetic model and experimental measurements, suggests that there is potential for increasing CAT yield in CFPS as well as CFPS performance. 
The theoretical maximum yield of CAT was 0.35 for an unconstrained case and 0.225 for a transcription/translation constrained case. 
Knockouts of \textit{gnd} and phosphate acetyltransferase, show carbon can be diverted away from acetate and potentially toward CAT or other proteins of interest expressed in CFPS. 
The other limitations to address in CFPS would be to enhance the transcription and translation rates since the protein of interest to be expressed is ultimately bounded by these kinetic rates.    
Li et al. have increased productivity of firefly lucifease by 5-fold in CFPS systems by adding and adjusting factors that affect transcription and translation such as elongation factors, ribosome recyclcing factor, release factors, chaperones, BSA and tRNAs \cite{2014_li_PlosOne}. 
Underwood et al. has also shown the increase in ribosome levels does not significantly increase protein yields and rates; however, adding elongation factors increased yields by 23\% at 30 minutes\cite{2005_underwood_biotech}.
In addition to improving CFPS performance, Jewett et al. has showed that oxidative phosphorylation operates in cell-free systems and knocking out oxidative phosphorylation reactions is detrimental to protein yield \cite{2008_jewett_molsysbiol}. 
However, it is inconclusive how much oxidative phosphorylation activity there is compared to \textit{in vivo} systems and both of our models suggest oxidative phosphorylation is vital to CAT production. 
Thus, this is a potential place for improvement to optimize CFPS for better performance and protein yield.   

%The ensemble of models captured the dynamic distribution of metabolic flux.
%The ensemble of dynamic models predicted that most of the carbon flux going through glycolysis and the TCA cycle,
%while the constraint-based approach predicted most of the carbon flux went through the Entner-Doudoroff pathway and only part of glycolysis.
% We present the first dynamic, cell-free model of \textit{E. coli} metabolism and biosynthesis at this scope.
% While the early models of Shuler and coworkers achieved large-scale, dynamic descriptions of single cells \citep{1984_domach_shuler_BiotechBioeng_01}, and the stoichiometric models associated with the FBA approach are computationally efficient amd widespread \citep{2009_feist_palsson_NatRevMicrobio}, none have yet been able to construct genome-scale kinetic models of cell-free \textit{E. coli} metabolism.
% This study should provide a foundation for genome-scale, dynamic modeling of cell-free \textit{E. coli} metabolism, toward industrial-scale biosynthetic production.
%However, the low sensitivities to glucose and oxygen only apply in the specific case of the parameter sets that were studied.
%When other species' initial conditions and associated parameters are different, glucose and oxygen could become more important to CAT production.

The cell-free model ensemble described the training data with the exception of some of the amino acids.
Specifically, adding more reactions that consume amino acids would improve the model's ability to predict those that show a decrase in the experimental data.
Also, including specific transcription and translation steps for CAT would allow us to more accurately model the complexity and the resource cost of protein synthesis.
Another area for future work is to more thoroughly sample parameter space.
For the metabolites in the dataset, initial conditions were fixed at the initial data values.
All other parameters were varied in a manner so as to best fit the dataset.
However, the resulting ensemble may not represent every biological or practical possibility.
In a different region of parameter space, the system could behave differently, including the flux distribution through the network, the accuracy and spread of ensemble fits, the relative sensitivities, and the yield as a percentage of the theoretical maximum.
Testing the model under a variety of conditions could strengthen or challenge the findings of this study.
Further experimentation could also be used to gain a deeper understanding of model performance under a variety of conditions.
Specifically, CAT production performed in the absence of amino acids could inform the system's ability to manufacture them, while experimentation in the absence of glucose or oxygen could shed light on how important they are to protein synthesis, and under which conditions.
Finally, the approach should be extended to other protein products.
CAT is only a test protein used for model identification; the modeling framework, and to some extent the parameter values, should be protein agnostic.
An important extension of this study would be to apply its insights to other protein applications, where possible.

\clearpage

\section*{Materials and Methods}

\subsection*{Formulation and solution of the model equations}
We used ordinary differential equations (ODEs) to model the time evolution of metabolite ($x_{i}$) and scaled enzyme abundance ($\epsilon_{i}$) in hypothetical cell-free metabolic networks:
\begin{eqnarray}
	\frac{dx_{i}}{dt} & = & \sum_{j=1}^{\mathcal{R}}\sigma_{ij}r_{j}\left(\mathbf{x},\mathbf{\epsilon},\mathbf{k}\right)\qquad{i=1,2,\hdots,\mathcal{M}}\\
	\frac{d\epsilon_{i}}{dt} & = & -\lambda_{i}\epsilon_{i}\qquad{i=1,2,\hdots,\mathcal{E}}
\end{eqnarray}where $\mathcal{R}$ denotes the number of reactions, $\mathcal{M}$ denotes the number of metabolites and $\mathcal{E}$ denotes the number of enzymes in the model.
The quantity $r_{j}\left(\mathbf{x},\mathbf{\epsilon},\mathbf{k}\right)$ denotes the rate of reaction $j$.
Typically, reaction $j$ is a non-linear function of metabolite and enzyme abundance, as well as unknown kinetic parameters $\mathbf{k}$ ($\mathcal{K}\times{1}$).
The quantity $\sigma_{ij}$ denotes the stoichiometric coefficient for species $i$ in reaction $j$.
If $\sigma_{ij}>0$, metabolite $i$ is produced by reaction $j$.
Conversely, if $\sigma_{ij}<0$, metabolite $i$ is consumed by reaction $j$, while $\sigma_{ij}=0$ indicates metabolite $i$ is not connected with reaction $j$.
Lastly, $\lambda_{i}$ denotes the scaled enzyme degradation constant.
The system material balances were subject to the initial conditions $\mathbf{x}\left(t_{o}\right)=\mathbf{x}_{o}$ and $\mathbf{\epsilon}\left(t_{o}\right)=\mathbf{1}$ (initially we have 100\% cell-free enzyme abundance).

The reaction rate was written as the product of a kinetic term ($\bar{r}_{j}$) and a control term ($v_{j}$), $r_{j}\left(\mathbf{x},\mathbf{k}\right)=\bar{r}_{j}v_{j}$.
In this study, we used either saturation or mass action kinetics.
The control term $0\leq v_{j}\leq 1$ depended upon the combination of factors which influenced rate process $j$.
For each rate, we used a rule-based approach to select from competing control factors.
If rate j was influenced by $1,\dots,m$ factors, we modeled this relationship as
$v_{j}=\mathcal{I}_{j}\left(f_{1j}\left(\cdot\right),\hdots,f_{mj}\left(\cdot\right)\right)$
where $0\leq f_{ij}\left(\cdot\right)\leq 1$ denotes a regulatory transfer function quantifying the influence of factor $i$ on rate $j$.
The function $\mathcal{I}_{j}\left(\cdot\right)$ is an integration rule which maps the output of regulatory transfer functions into a control
variable. Each regulatory transfer function took the form:
\begin{equation}\label{eqn:control-factor}
	f_{ij}\left(\mathcal{Z}_{i},k_{ij},\eta_{ij}\right)=k_{ij}^{\eta_{ij}}\mathcal{Z}_{i}^{\eta_{ij}}/\left({1 + k_{ij}^{\eta_{ij}}\mathcal{Z}_{i}^{\eta_{ij}}}\right)
\end{equation}where $\mathcal{Z}_{i}$ denotes the abundance factor $i$, $k_{ij}$ denotes a gain parameter, and $\eta_{ij}$ denotes a cooperativity parameter.
In this study, we used $\mathcal{I}_{j}\in\left\{mean\right\}$ \cite{pr3010138}. If a process has no modifying factors, $v_{j}=1$.
We used multiple saturation kinetics to model the reaction term $\bar{r}_{j}$:
\begin{equation}\label{eqn:rate-bar}
	\bar{r}_{j}=k_{j}^{max}\epsilon_{i}\left(\prod_{s\in{m_{j}^{-}}}\frac{x_{s}}{K_{js} + x_{s}}\right)
\end{equation}
where $k_{j}^{max}$ denotes the maximum rate for reaction $j$, $\epsilon_{i}$ denotes the scaled enzyme activity which catalyzes reaction $j$, and
$K_{js}$ denotes the saturation constant for species $s$ in reaction $j$.
The product in Equation~\eqref{eqn:rate-bar} was carried out over the set of \textit{reactants} for reaction $j$ (denoted as $m_{j}^{-}$).

We added regulation to the network as informed by literature, for a total of 17 interactions.
PEP was modeled as an inhibitor for phosphofructokinase \cite{2010_kotte_MolSystBiol,2011_cabrera_JBiolChem}, PEP carboxykinase \cite{2010_kotte_MolSystBiol}, PEP synthetase \cite{2010_kotte_MolSystBiol,1973_chulavatnatol_JBiolChem}, isocitrate dehydrogenase \cite{2010_kotte_MolSystBiol,2007_ogawa_JBacteriol}, and isocitrate lyase/malate synthase \cite{2010_kotte_MolSystBiol,2007_ogawa_JBacteriol,1988_mackintosh_BiochemJ}, and as an activator for fructose-biphosphatase \cite{2010_kotte_MolSystBiol,2000_donahue_JBacteriol,2006_hines_JBiolChem,2007_hines_JBiolChem}.
AKG was modeled as an inhibitor for citrate synthase \cite{2010_kotte_MolSystBiol,1994_pereira_JBiolChem,1983_robinson_FEBSLett} and isocitrate lyase/malate synthase \cite{2010_kotte_MolSystBiol,1988_mackintosh_BiochemJ}.
3PG was modeled as an inhibitor for isocitrate lyase/malate synthase \cite{2010_kotte_MolSystBiol,1988_mackintosh_BiochemJ}.
FDP was modeled as an activator for pyruvate kinase \cite{2010_kotte_MolSystBiol,2010_zhu_Biochimie} and PEP carboxylase \cite{2010_kotte_MolSystBiol,1972_wohl_JBiolChem}.
Pyruvate was modeled as an inhibitor for pyruvate dehydrogenase \cite{2010_kotte_MolSystBiol,2007_kale_JBiolChem,2002_arjunan_Biochemistry} and as an activator for lactate dehydrogenase \cite{2008_okino_ApplMicrobiolBiotechnol}.
Acetyl CoA was modeled as an inhibitor for malate dehydrogenase \cite{2010_kotte_MolSystBiol}.

\subsection*{Generation of model ensemble}
We generated an ensemble of 100 diverse parameter sets via a Markov chain Monte Carlo random walk.
Beginning with a single parameter set as a starting point, we calculated its cost function, equal to the sum-squared-error between experimental data and model predictions:
\begin{equation}\label{eqn:cost-function}
    cost=\sum_{i=1}^{\mathcal{D}}\left(w_i\sum_{j=1}^{\mathcal{T}_i}abs\left(x_{ij}^{data}-x_{i}^{sim}|_{t(j)}\right)\right)
\end{equation}
where $\mathcal{D}$ denotes the number of datasets, equal to 37, $w_i$ denotes the weight of the $i$th dataset, $\mathcal{T}_i$ denotes the number of timepoints in the $i$th dataset, $t(j)$ denotes the $j$th timepoint, $x_{ij}^{data}$ denotes the value of the $i$th dataset at the $j$th timepoint, and $x_{i}^{sim}|_{t(j)}$ denotes the simulated value of the metabolite corresponding to the $i$th dataset, interpolated to the $j$th timepoint.
We then perturbed model parameters:
\begin{equation}\label{eqn:parameter-perturbation}
    k_i^{new}=k_i*exp(a\,r_i)\qquad{i=1,2,\hdots,\mathcal{P}}
\end{equation}
where $\mathcal{P}$ denotes the number of parameters, equal to 815, which includes 163 rate constants, 163 enzyme degradation rate constants, 455 saturation constants, and 34 control parameters, $k_i^{new}$ denotes the new value of the $i$th parameter, $k_i$ denotes the current value of the $i$th parameter, $a$ denotes a distribution variance, and $r_i$ denotes a random sample from the normal distribution.
For each newly generated parameter set, we re-solved the balance equations and calculated the cost function.
All sets with a lower cost than the previous set, and some with higher cost, were added to the ensemble.
After generating 12,437 sets, we selected 100 sets with minimal correlation to each other so as to avoid over-sampling any region of parameter space.
The original 12,437-set ensemble had a mean Pearson correlation coefficient [REFERENCE NEEDED?] of 0.94 between pairs of sets, while the 100-set ensemble had a mean Pearson correlation coefficient of 0.83 between pairs of sets.

\subsection*{Local sensitivity analysis}
We determined the reactions most important to protein production by computing the local sensitivity of CAT concentration to each rate constant in the network.
Each sensitivity index was formulated as:
\begin{equation}\label{eqn:CAT-sensitivity}
\begin{split}
    S_{ij}=norm(CAT(p_i,p_j,t)-CAT(\alpha*p_i,\alpha*p_j,t))\qquad{i,j=1,2,\hdots\mathcal{P}}\\
\end{split}
\end{equation}
where $S_{ij}$ denotes the sensitivity of CAT production to the $i$th and $j$th parameters, $CAT(p_i,p_j,t)$ denotes CAT concentration as a function of time and the $i$th and $j$th parameters, $\alpha$ denotes the perturbation factor, equal to 1.01, and $\mathcal{P}$ denotes the number of rate constants, equal to 163.
In calculating the pairwise sensitivities, each parameter was perturbed by 1\%; first-order sensitivities ($i$ = $j$) were subject to two 1\% perturbations, equivalent to a perturbation of 2.01\%.
The array of sensitivity indices was organized into a clustergram (Fig.~\ref{fig:Sensitivity}, top) with both axes corresponding to the 163 rate constants rearranged from most sensitive (top, right) to least sensitive (bottom, left).

Likewise, we determined the reactions most important to the system as a whole by computing the local sensitivity of the species for which data exist to each rate constant in the network.
Each sensitivity index was formulated as:
\begin{equation}\label{eqn:system-sensitivity}
\begin{split}
    S_{ij}=norm(X(p_i,p_j,t)-X(\alpha*p_i,\alpha*p_j,t))\qquad{i,j=1,2,\hdots\mathcal{P}}\\
\end{split}
\end{equation}
where $S_{ij}$ denotes the sensitivity of the system state to the $i$th and $j$th parameters, and $X(p_i,p_j,t)$ denotes the system state, an array consisting of the concentration of every species for which data exist as a function of time and the $i$th and $j$th parameters.
% $\alpha$ denotes the perturbation factor, equal to 1.01, and $\mathcal{P}$ denotes the number of rate constants, equal to 163.
This array of sensitivity indices was also organized into a clustergram (Fig.~\ref{fig:Sensitivity}, bottom).

\subsection*{Sequence specific FBA and calculation of CAT yield}
The yield on CAT production was calculated for each case as a ratio of carbon produced as CAT to carbon consumed as reactants (glucose and amino acids):
\begin{equation}\label{eqn:yield-definition}
	Yield=\frac{\Delta CAT\;C_{CAT}}{\sum_{i=1}^{\mathcal{R}}\max(\Delta m_{i},0)\;C_{m_i}}
\end{equation}
where $\Delta CAT$ denotes the amount of CAT produced, $C_{CAT}$ denotes carbon number of CAT, $\mathcal{R}$ denotes the number of reactants, $\Delta m_{i}$ denotes the amount of the $i$th reactant consumed, never allowed to be negative, and $C_{m_i}$ denotes the carbon number of the $i$th reactant.
Because no data was available for arginine or glutamate, these reactants were left out of all three calculations.
In the experimental case and the best-fit set case, yield was calculated by setting $\Delta CAT$ equal to the final minus the initial CAT concentration and setting $\Delta m_{i}$ equal to the initial minus the final reactant concentration.
The theoretical yield was calculated using flux balance anaylsis (FBA) with a sequence-specific based analysis on CAT.
The sequence specific FBA \cite{2002_allen_palsson} problem was formulated as:
\begin{equation}\nonumber
 \begin{multlined}
	\qquad \qquad \qquad \max_{\boldsymbol{w}}{} \! \left( w_{obj} = \mathbf{\boldsymbol{\theta}}^T \boldsymbol{w} \right) \\
	\mathrm{Subject \; to:}
	 \; \; \mathbf{S}\mathbf{w}=\mathbf{0} \\
\alpha_i \leq w_i \leq \beta_i  \qquad i=1,2,\hdots,\mathcal{R}
 \end{multlined}
\end{equation}
where $\mathbf{S}$ denotes the stoichiometric matrix, $\mathbf{w}$ denotes the unknown flux vector, $\boldsymbol{\theta}$ denotes the objective selection vector
and $\alpha_i$ and $\beta_i$ denote the lower and upper bounds on flux $w_{i}$, respectively.
The objective $w_{obj}$ was to maximize the specific rate of CAT formation.
The specific glucose uptake rate was constrained to allow a maximum flux of 40 mM/hr according to experimental data; the specific amino acid uptake rates were bound to allow a maximum flux of 30 mM/hr, but did not reach this maximum flux.
The transcription and translation template reactions were added to the metabolic network and are based off sequence specific analysis \cite{2002_allen_palsson} involving transcription initiaion, transcription, mRNA degradation, translation initiation, translation, and tRNA charging. 
The flux balance analysis problem was solved using the GNU Linear Programming Kit (v4.52) \cite{GLPK}.
The solution flux vector was used to calculate the theoretical carbon yield of CAT.
Glucose, oxygen, and amino acids were modeled as being imported into the system, whereas CAT synthesis and metabolite byproduct formation was modeled as an export from the system.
The rest of the network followed a pseudo steady-state asusmption where all other metabolites were not allowed to accumulate; thus, the network could be solved by linear programming.

The transcription rate was constrained as:
\begin{equation}\nonumber
	w_{tx} = RNAP\frac{v_{RNAP}}{l_{mRNA}}\left(\frac{Gene}{km+Gene}\right)P
\end{equation}
where $RNAP$ is the concentration of RNA polymerase, $v_{RNAP}$ is the elongation rate (nucleotides/hr) by the RNA polymerase, $l_{mRNA}$ is the number of nucleotides in the mRNA, $Gene$ is the gene concentration, $km$ is the plasmid saturation coefficient, and $P$ is the promoter activity.
The mRNA and protein sequence of CAT was determined from literature.

The promoter activity was formulated following Moon et al. for synthetic circuits as:
\begin{equation}\nonumber
	P = \frac{K_{1}}{1 + K_{1}}
\end{equation}
where $K_{1}$ represents the state of T7 RNA polymerase binding. 

The translation rate was constrained as:
 \begin{equation}\nonumber
	w_{tl} = K_{P}Ribo\frac{v_{Ribo}}{l_{protein}}[mRNA_{ss}]
\end{equation}
where $K_{P}$ is the polysome amplification constant, $Ribo$ is the ribosome concentration,  $v_{Ribo}$ is the elongation rate (amino acids/hr) of the ribosome, $l_{protein}$ is the number of amino acids in the protein of interest, and $mRNA_{ss}$ is the mRNA concentration at steady state determined by the transcription rate divided by the degradation rate of mRNA.  

%\begin{equation}\label{eqn:yield}
%	Yield=\frac{w_{CAT}\;C_{CAT}}{\sum_{i=1}^{\mathcal{R}}\max(w_{i},0)\;C_{i}}
%\end{equation}
%where $w_{CAT}$ denotes the CAT export flux and $w_{i}$ denotes the import flux of the $i$th substrate.
An ensemble of 100 sets of flux distributions was calculated for three different cases, unconstrained, constrained by transcription and translation (TX/TL) rates, and  constrained by TX/TL rates and experimental data. 
For the unconstrained case, all rates were left unbounded, except for the specific glucose uptake rate. 
An ensemble of flux distributions was calculated by randonmly sampling the maximum specific glucose uptake rate from 30 to 40 mM/hr determined from experimental data.
For the case constrained by TX/TL rates, an ensemble was generated by randomly sampling RNAP polymerase levels, ribosome levels, and elongation rates in a physiological range determined from literature.
RNA polymerase levels were sampled between 60 and 80 nM, ribosome levels between 7 and 16 \textmu M, the elongation rate by RNA polymerase between 20 and 30 nts/sec, and the elongation rate by ribosomes between 1.5 and 3 AA/sec \cite{2005_underwood_biotech, 2016_garamella_acssynbio}.
For the case constrained by TX/TL rates and experimental data, an ensemble was generated by randomly sampling RNAP polymerase levels, ribosome levels, and elongation rates in a physiological range determined from literature. 
The lower and upper bound constraints were randomly sampled in the physiological range of the experimental noise where data was available, except for CAT flux, which was determined from the transcription and translation rates. 
\clearpage

%Need to talk more about biochemical benefits and importance for of biochemical problems

\section*{Funding}
This study was supported by a National Science Foundation Graduate Research Fellowship (DGE-1333468) to N.H and by an award from the US Army and Systems Biology of Trauma Induced Coagulopathy (W911NF-10-1-0376) to J.V for the support of M.V.

\clearpage

\bibliography{References_v1}

\clearpage

% Figures and captions go here ...
\begin{figure}[ht]
\centering
\includegraphics[width=1.00\textwidth]{./Figures/CarbonBoth.pdf}
\caption{Central carbon metabolism in the presence (top) and absence (bottom) of allosteric control, including glucose (substrate), CAT (product), and intermediates, as well as total concentration of energy species. Best-fit parameter set (orange line) versus experimental data (points). 95\% confidence interval (blue shaded region) and 95\% confidence interval of the mean (gray shaded region) over the ensemble of 100 sets.}
\label{fig:CarbonBoth}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=1.00\textwidth]{./Figures/CarbonBoth_png.pdf}
\caption{Central carbon metabolism in the presence (top) and absence (bottom) of allosteric control, including glucose (substrate), CAT (product), and intermediates, as well as total concentration of energy species. Best-fit parameter set (orange line) versus experimental data (points). 95\% confidence interval (blue shaded region) and 95\% confidence interval of the mean (gray shaded region) over the ensemble of 100 sets.}
\label{fig:CarbonBoth_png}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=1.00\textwidth]{./Figures/Energy.pdf}
\caption{Energy species and energy totals by base in the presence of allosteric control. Best-fit parameter set (orange line) versus experimental data (points). 95\% confidence interval (blue shaded region) and 95\% confidence interval of the mean (gray shaded region) over the ensemble of 100 sets.}
\label{fig:Energy}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=1.00\textwidth]{./Figures/Amino.pdf}
\caption{Amino acids in the presence of allosteric control. Best-fit parameter set (orange line) versus experimental data (points). 95\% confidence interval (blue shaded region) and 95\% confidence interval of the mean (gray shaded region) over the ensemble of 100 sets.}
\label{fig:Amino}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=1.00\textwidth]{./Figures/BoxPlot-KCFM.pdf}
\caption{Log of cost function across 37 datasets for data-trained ensemble (blue) and randomly generated ensemble (red, gray background). Median (bars), interquartile range (boxes), range excluding outliers (dashed lines), and outliers (circles) for each dataset. Median across all datasets (large bar overlaid).}
\label{fig:BoxPlot}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=0.76\textwidth,trim=0 28 615 40,clip]{./Figures/Sensitivity.pdf}
\caption{Normalized first-order and pairwise sensitivities of CAT production (top) and system state (bottom) to rate constants.}
\label{fig:Sensitivity}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=0.95\textwidth]{./Figures/CAT_prod_yield.pdf}
\caption{Sequence specific flux balance analysis of CAT production and yield. A. 95\% confidence interval of the ensemble (light blue region) for CAT concentration versus time. B. Theoretical maximum carbon yield of CAT calcualted by ssFBA for three different cases: unconstrained except for glucose uptake (black), constrained by transcription and translation rates (grey), and constrained by transcription, translation rates and experimental measurements where available (light grey). Error bars represent standard deviation of the ensemble.}
\label{fig:CATProdYield}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=0.8\textwidth]{./Figures/CAT_flux_final.pdf}
\caption{Flux profile for glycolysis, pentose phosphate pathway, Entner-Doudoroff pathway, TCA cycle, NADPH/NADH transfer, and oxidative phosphorylation. Sequence specific FBA flux value (mean ± standard deviation) across ensemble for 1 hr, normalized to glucose uptake flux. Flux distribution for three different cases: unconstrained except for glucose uptake (top row), constrained by transcription and translation rates (second row), and constrained by transcription, translation rates and experimental measurements where available (bottom row).}
\label{fig:Network}
\end{figure}

\clearpage

% Supplemental figures -
% Set the S-
\renewcommand\thefigure{S\arabic{figure}}
\renewcommand\thetable{T\arabic{table}}
\renewcommand\thepage{S-\arabic{page}}
\renewcommand\theequation{S\arabic{equation}}

% Reset the counters -
\setcounter{equation}{0}
\setcounter{table}{0}
\setcounter{figure}{0}
\setcounter{page}{1}

% Supplemental figures go here ...
%\begin{figure}[ht]
%\centering
%\includegraphics[width=1.00\textwidth]{./figs/<Filename>.pdf}
%\caption{Captiontext goes here}
%}\label{fig:<label_name>}
%\end{figure}

\end{document}
\grid
\grid
